index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class='navContainer'>
+        <div class="wrapper">
+            <nav>
+                <div class="logo">
+                    Ivan
+                </div>
+                <ul class="navItems">
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a class="navBtnMessage" href="#">
+                            <img class="messageBtn" src="images/mail.png" alt="">
+                        </a>
+                    </li>
+                </ul>
+                <ul class="navMenu">
+                    <li>
+                        <a class="navBtnMenu" href="#">
+                            <img class="menuBtn" src="images/menuUnion.png" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </nav>   
+        </div>
+    </div>
+
+    <div class="headline-container">
+        <div class="wrapper">
+            <header>
+                <div class="headlineContent">
+                    <h1>Hey there. I'm Ivan.</h1>
+                    <span>I’m a Software Developer</span>
+                    <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                    <div class="headlineSocial">
+                        <a href="#">
+                            <img src="images/gitHub.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/linkedin.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/angelist.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/twitter.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/medium.png" alt="">
+                        </a>
+                        
+                    </div>
+                </div>
+
+                <div class="headlineImage">
+                    <img src="images/illustration.png" alt="">
+                </div>
+            </header>
+        </div>
+    </div>
+</body>
```

Comment:
- No images are working in your project. 
- [ ] `First:`  There should be a menu-hamburger icon.
- [ ] `Second:` There should be a `background image`. Kindly use `background-image` property of  CSS for this section.
- [ ]  `Third:` There should be `social media` icons with links mentioned  in the template.
- ![image](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/60083437/bfe2bb9d-1ac0-4a77-9478-3d36580fe505)

- I can see you have tried to use images using `img` tag `<img src="images/....." alt="">`, but there is no `images` folder in your repository. May be that is the issue the images and icons are not working. Kindly fix this issue.